docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
 
   mongo:
     image: mongo:3.4
-    container_name: mongo
+    container_name: st2mongo
```

Comment:
Can't we just remove the `container_name` entirely? It is much simpler. This still have a problem when we want to boot more than two sets of this compose in the same docker machine.